javatests/google/registry/flows/domain/DomainCreateFlowTest.java

```diff
@@ -437,6 +437,18 @@ public void testSuccess_generalAvailability_ignoresEncodedSignedMark() throws Ex
   }
 
   @Test
+  public void testSuccess_superuser_ignoresEncodedSignedMark() throws Exception {
```

Comment:
This test is not trying to verify ignoring a signed mark mismatch. It's testing that validation on the signed mark is not happening if ran as a superuser. I setup the mismatch as it was an easy way to fail the validation if it happened to be called.